react.js

```diff
@@ -1,11 +1,17 @@
 module.exports = {
   extends: [
     'eslint-config-airbnb-typescript',
+    'plugin:react/recommended',
```

Comment:
I'm not 100% sure why this was needed. without this all `react/*` rules were undefined. I thought the airbnb config pulls this in indirectly